presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PayloadJoinOptimizer.java

```diff
@@ -343,15 +343,15 @@ private PlanNode transformJoin(JoinNode joinNode, JoinContext context)
             }
 
             ProjectNode projectNode = new ProjectNode(planNodeIdAllocator.getNextId(), joinNode, assignments.build());
-            List<VariableReferenceExpression> resultOutputCols = Stream.concat(planNode.getOutputVariables().stream(), projectNode.getOutputVariables().stream()).collect(toImmutableList());
+            List<VariableReferenceExpression> resultOutputCols = Stream.concat(projectNode.getOutputVariables().stream(), planNode.getOutputVariables().stream()).collect(toImmutableList());
```

Comment:
Can we name the sides appropriately - like keysNode and payLoadNode so it will be obvious for the reader?

Reply:
+1